Formula/thrift.rb

```diff
@@ -46,15 +47,24 @@ def install
       --prefix=#{prefix}
       --libdir=#{lib}
       --with-openssl=#{Formula["openssl@1.1"].opt_prefix}
-      --without-erlang
-      --without-haskell
```

Comment:
Any reason why we're removing this?

Reply:
Whoops, didn't see those were being removed!  Sorry I merged already.  I can add back as a followup if needed, as I don't know why we'd remove these either.

Reply:
Actually, `--without-erlang` was just moved down, but `--without-haskell` seems gone now.

Reply:
Yes, I was referring out `--without-haskell`?

Reply:
@danielnachun and @carlocab `haskell` was removed from thrift, same as common lisp:

- https://github.com/apache/thrift/tree/master/lib
- https://github.com/apache/thrift/tree/v0.16.0/lib
- https://issues.apache.org/jira/browse/THRIFT-5347

Reply:
for future references, the list (and its ordering) was copied from https://github.com/apache/thrift/blob/b664cfe2533e4bbf00fd5e7e0211bf7161ee2a04/configure.ac#L115-L133

Reply:
I see, so `--without-haskell` was just being ignored.  I think everything is okay then.

Reply:
affirmative, `--without-haskell` would have been ignored (unused flag) for 0.16.0 and `head` building.